### PR TITLE
july 2021 newsletter

### PR DESCRIPTION
**Pull request by davidwyld**
Reviews welcome! Will send tomorrow morning.

### CODE REVIEW

**Review by Robadob — COMMENTED** [review]
Couple of places mention "Discussion topic" having a separate submission deadline for septembeRSE. This date has since passed, so maybe just strip out the two comments about discussion topic.

No other problems stand out to me.

**Comment by davidwyld** [comment]
will do, thanks @Robadob !

File: _newsletters/2021-07-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+The [Conference of Research Software Engineers](https://septembrse.society-rse.org/) is returning this year with a fully online event.
+
+6 - 30th September 2021
+
+> On Monday 6th September 2021, the RSE Community will reunite for SeptembRSE, the Fifth Conference of Research Software Engineers. The conference is a chance for anyone, whether you consider yourself a Research Software Engineer or not, to meet and discuss the development and recognition of research software.
+
+> This year the entire conference has moved online. The flexibility achievable with online events allows the sessions to be spread out so that they occupy the whole of September. All sessions will be streamed, recorded and published, meaning that you will have a whole month of material to explore throughout September at a time that is best for you!
+
+> Submissions are now being taken for; Talks, Posters, Walkthroughs, Workshops, Panels and Discussions. So if you have an idea and want to get involved with the wider RSE community submit your proposal now!
```
Comment (bobturneruk) [inline _newsletters/2021-07-07-newsletter.md]:
```suggestion
```

File: _newsletters/2021-07-07-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+[Submission Deadline](https://septembrse.society-rse.org/calls/): 2nd July 2021
```
Comment (bobturneruk) [inline _newsletters/2021-07-07-newsletter.md]:
```suggestion
```

File: _newsletters/2021-07-07-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+> We have a few ideas for running our live sessions and commissioning content for the Camp, but we’re asking our community about what would be useful to hear/read about and learn from around the use of spreadsheets in research, what works, and taking the first steps into other computation techniques. 
+
+[Submit ideas by the end of July](https://www.software.ac.uk/news/research-software-camp-beyond-spreadsheet-call-ideas)
+
+#### FAIR Principles for Research Software
+The [Research Data Alliance](https://www.rd-alliance.org/) has developed a set of principles for research software inspired by the FAIR principles for research data (findable, accessible, interoperable, reusable).
+
+The community review period for the guiding document is open until the 11th July (here)[https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs]
```
Comment (bobturneruk) [inline _newsletters/2021-07-07-newsletter.md]:
```suggestion
The community review period for the guiding document is open until the 11th July [here](https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs)
```

File: _newsletters/2021-07-07-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+EPSRC have announced a call for £4.5M funding to adapt or maintain existing software used by research communities.
+
+>Your project must be within the remit of EPSRC.
+>You can use the funding for:
+> * development and re-engineering of existing software
+> * maintenance activities
+> * activities that widen participation in development and maintenance.
+
+Please contact the [RSE Team][rses] to discuss.
```
Comment (bobturneruk) [inline _newsletters/2021-07-07-newsletter.md]:
```suggestion

The Research Software Engineering (RSE) Team (<https://rse.shef.ac.uk/>) are delighted to see the new call from the EPSRC: “Software for research communities” (<https://www.ukri.org/opportunity/software-for-research-communities>).
It is something we’ve been advocating for for some time and it’s great to see it finally happening! The call specifically is specifically looking for inclusion of RSEs as part of the project delivery and we would be very happy to discuss with PIs about inclusion of RSEs from our team.

Researchers can cost our engineers in funding applications at a proportion of full time equivalent, bringing in software engineering expertise without the need to hire a dedicated RSE. If this is of interest to you in the context of this call, or any other funding application, please contact <rse@sheffield.ac.uk> at the earliest opportunity to talk about the project. We can provide useful advice even at an early stage, and we need to know well in advance about applications being submitted so we can do our best to arrange engineers with the best skills and experience when the project is (fingers-crossed!) funded.
```

File: _newsletters/2021-07-07-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+* [Machine Learning in the RSE Team [RSE Sheffield Blog]](https://rse.shef.ac.uk/blog/2021-06-18-ml-intro-vid/) - A brief intro to where Machine Learning fits into the work of the University of Sheffield RSE team.
+* [Impact factor abandoned by Dutch university in hiring and promotion decisions [Nature]](https://www.nature.com/articles/d41586-021-01759-5) - "Faculty and staff members at Utrecht University will be evaluated by their commitment to open science."
+
+### Training
+**[High Performance Computing for Healthcare](https://www.eventbrite.co.uk/e/high-performance-computing-for-healthcare-tickets-144650454403)**
+
+Taking place over 2 days (12th & 14th July) in partnership with the [N8CIR Universities](https://n8cir.org.uk/), [Sheffield Bioinformatics Core](https://sbc.shef.ac.uk/) and [Sheffield RSE Team][rses] will be delivering training on using HPC for genomics research.
+
+[Sign up here](https://www.eventbrite.co.uk/e/high-performance-computing-for-healthcare-tickets-144650454403).
```
Comment (bobturneruk) [inline _newsletters/2021-07-07-newsletter.md]: Has the sign up closed? cc @willfurnass

**Review by bobturneruk — APPROVED** [review]
Awesome sauce. Few suggestions.

File: _newsletters/2021-07-07-newsletter.md (hunk 5 above)
Comment (willfurnass) [reply _newsletters/2021-07-07-newsletter.md]: Yes, let's remove this here and mention in the next newsletter after the event

**Comment by davidwyld** [comment]
Thanks for the feedback all!

**Pull request merged**